pkgs/development/tools/cargo-flamegraph/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, rustPlatform
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-flamegraph";
+  version = "0.1.13";
+
+  src = fetchFromGitHub {
+    owner = "ferrous-systems";
+    repo = "flamegraph";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
The names differ -- `pname` is "cargo-flamegraph".

Reply:
Right, thanks for noticing.